_i18n/en/resources/developer-guides/daemon-rpc.md

```diff
@@ -563,270 +783,1636 @@ $ curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","me
+Gives an estimation on fees per kB.
+
+Alias: *None*.
 
 Inputs:
 
-* *key_images* - string list; List of key image hex strings to check.
+* *grace_blocks* - unsigned int; Optional
 
 Outputs:
 
-* *spent_status* - unsigned int list; List of statuses for each image checked. Statuses are follows: 0 = unspent, 1 = spent in blockchain, 2 = spent in transaction pool
+* *fee* - unsigned int; Amount of fees estimated per kB in @atomic-units
 * *status* - string; General RPC error code. "OK" means everything looks good.
+* *untrusted* - boolean;
 
 Example:
 
 ```
-$ curl -X POST http://127.0.0.1:18081/is_key_image_spent -d '{"key_images":["8d1bd8181bf7d857bdb281e0153d84cd55a3fcaa57c3e570f4a49f935850b5e3","7319134bfc50668251f5b899c66b005805ee255c136f0e1cecbb0f3a912e09d4"]}' -H 'Content-Type: application/json'
+$ curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"get_fee_estimate"}' -H 'Content-Type: application/json'
 
 {
-  "spent_status": [1,2],
-  "status": "OK"
+  "id": "0",
+  "jsonrpc": "2.0",
+  "result": {
+    "fee": 187610000,
+    "status": "OK",
+    "untrusted": false
+  }
 }
 ```
 
 
-### **/sendrawtransaction**
+### **get_alternate_chains**
 
-Broadcast a raw transaction to the network.
+Display alternative chains seen by the node.
 
-Inputs:
+Alias: *None*.
 
-* *tx_as_hex* - string; Full transaction information as hexidecimal string.
-* *do_not_relay* - boolean; Stop relaying transaction to other nodes (default is `false` now; in versions 0.11.1 it was `true`).
+Inputs: *None*.
 
 Outputs:
 
-* *status* - string; General RPC error code. "OK" means everything looks good. Any other value means that something went wrong.
-* *double_spend* - boolean;  Transaction is a double spend (`true`) or not (`false`).
-* *fee_too_low* - boolean; Fee is too low (`true`) or OK (`false`).
-* *invalid_input* - boolean; Input is invalid (`true`) or valid (`false`).
-* *invalid_output* - boolean; Output is invalid (`true`) or valid (`false`).
-* *low_mixin* - boolean; Mixin count is too low (`true`) or OK (`false`).
-* *not_rct* - boolean; Transaction is not a ring transaction (`true`) or a ring transaction (`false`).
-* *not_relayed* - boolean; Transaction was not relayed (`true`) or relayed (`false`).
-* *overspend* - boolean; Transaction uses more money than available (`true`) or not (`false`). 
-* *reason* - string; Additional information. Currently empty or "Not relayed" if transaction was accepted but not relayed.
-* *too_big* - boolean; Transaction size is too big (`true`) or OK (`false`).
-
-
-Example (No return information included here.):
+* *chains* - array of chains, the following structure:
+  * *block_hash* - string; the block hash of the first diverging block of this alternative chain.
+  * *difficulty* - unsigned int; the cumulative difficulty of all blocks in the alternative chain.
+  * *height* - unsigned int; the block height of the first diverging block of this alternative chain.
```

Comment:
I think it should be the block height of the last non diverging block... not 100% sure. Unless it's the chain height at the fork, in which case it happens to match the block height of the first diverging block...

Reply:
@moneromooo-monero who could confirm?

Reply:
I would say it's the first diverging block. If i look at the returned block height on a block explorer, it's a different block hash.